### ISSUE TITLE
`az vm open-port` fails to accept list of ports

### ISSUE THREAD

**Issue by infogulch** [Azure/azure-cli#16037]
## Describe the bug

**Command Name**
`az vm open-port`

**Errors:**
```
BadRequestError: Security rule has invalid Port range. Value provided: 80,443. Value should be an integer OR integer range with '-' delimiter. Valid range 0-65535.
```

## To Reproduce:

- create azure vm
- `az vm open-port --name $vm -g $rg --port 80,443`

## Expected Behavior

`az vm open-port` succeeds and adds a new nsg rule which allows inbound network traffic to `$vm` on ports 80 & 443.

After this command failed, I tried to add it in the portal manually which worked as expected:
![image](https://user-images.githubusercontent.com/133882/100182617-a99ab480-2ea2-11eb-8627-a59f61357ccc.png)

## Environment Summary
```
Linux-4.19.128-microsoft-standard-x86_64-with-debian-buster-sid
Python 3.6.10
Installer: DEB

azure-cli 2.15.1
```

**Comment by yungezz** [comment]
hi @qwordy could you pls have a look? thanks

**Comment by qwordy** [comment]
@infogulch 
https://docs.microsoft.com/en-us/cli/azure/vm?view=azure-cli-latest#az_vm_open_port
It does support ",". Please specify one port or port range.
You can try
```
az vm open-port --name $vm -g $rg --port 80
az vm open-port --name $vm -g $rg --port 443
```

**Comment by infogulch** [comment]
@qwordy then what is that in the screenshot that I took?  Is this a bug report for the portal instead?

**Comment by qwordy** [comment]
I agree with you. It is easier to use if we can use only one command to open multiple ports. Mark this issue as a feature request.

**Comment by 08nholloway** [comment]
resolved in #17255
